src/components/common/Interactive.tsx

```diff
@@ -40,55 +26,35 @@ interface Props {
 
 const InteractiveBase = ({ onMove, onKey, ...rest }: Props) => {
   const container = useRef<HTMLDivElement>(null);
-  const hasTouched = useRef(false);
-  const [isDragging, setDragging] = useState(false);
   const onMoveCallback = useEventCallback<Interaction>(onMove);
   const onKeyCallback = useEventCallback<Interaction>(onKey);
 
-  // Prevent mobile browsers from handling mouse events (conflicting with touch ones).
-  // If we detected a touch interaction before, we prefer reacting to touch events only.
-  const isValid = (event: MouseEvent | TouchEvent): boolean => {
-    if (hasTouched.current && !isTouch(event)) return false;
-    if (!hasTouched.current) hasTouched.current = isTouch(event);
-    return true;
-  };
-
   const handleMove = useCallback(
-    (event: MouseEvent | TouchEvent) => {
-      // Prevent text selection
-      preventDefaultMove(event);
-
-      // If user moves the pointer outside of the window or iframe bounds and release it there,
-      // `mouseup`/`touchend` won't be fired. In order to stop the picker from following the cursor
-      // after the user has moved the mouse/finger back to the document, we check `event.buttons`
-      // and `event.touches`. It allows us to detect that the user is just moving his pointer
-      // without pressing it down
-      const isDown = isTouch(event) ? event.touches.length > 0 : event.buttons > 0;
-
-      if (isDown && container.current) {
+    (event: PointerEvent) => {      
+      if (container.current) {
         onMoveCallback(getRelativePosition(container.current, event));
-      } else {
-        setDragging(false);
       }
     },
     [onMoveCallback]
   );
 
   const handleMoveStart = useCallback(
-    ({ nativeEvent }: React.MouseEvent | React.TouchEvent) => {
+    ({ nativeEvent }: React.PointerEvent) => {
       const el = container.current;
+      if (!el) {
+        return;
+      }
 
-      // Prevent text selection
-      preventDefaultMove(nativeEvent);
-
-      // Interrupt "mousedown" call on mobiles
-      if (!isValid(nativeEvent) || !el) return;
+      // testing-library doesn't have setPointerCapture implemented
+      el.setPointerCapture && el.setPointerCapture(nativeEvent.pointerId);
```

Comment:
This simple line makes:
1) Multi touch support
2) All subsequent pointermove and pointerup events will be captured by this particular element, hence if the pointer is released outside the browser it still will be captured (see screencasts section in description)

More info: https://developer.mozilla.org/en-US/docs/Web/API/Element/setPointerCapture